tests/test_export.py

```diff
@@ -63,6 +64,38 @@ def test_export_datasets(archive, layer, geometry, data_archive, cli_runner):
             assert field_name[0:10] in output
 
 
+@pytest.mark.parametrize("crs", ["EPSG:4326", "EPSG:27200"])
+def test_export_with_transformed_crs(crs, data_archive, cli_runner):
```

Comment:
Here's the test checklist and my feeling about how covered it is:
- using dataset creation options ✅︎#990
- using layer creation options ✅︎#990
- gdal config options (via env) ✅#991
- existing directories, existing files, etc ✅︎#991
- maybe a set of common formats? (SHP, FGDB, GeoJSON, CSV) 🤔somewhat done #991
(tests now do at least one export to all of these formats, but they don't dig into the result.)
- error handling / error messages 🤔somewhat done #991
(the more we study individual drivers, the more we can detect their error conditions and raise more helpful errors. But so far error handling is very generic.)
- truncated/fixed/clashing field names (SHP in particular) ❌not done